src/styles/blocks/wrap.scss

```diff
@@ -0,0 +1,7 @@
+.wrap {
+  padding: 50px 0;
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 944px;
```

Comment:
don't hardcode max-width, your catalog container should work fine for any number of cards